spec/World_spec.lua

```diff
@@ -407,5 +413,27 @@ describe('World', function()
         assert.same({0,-3,1,1}, {world:getRect(a)})
       end)
     end)
+
```

Comment:
It's been a long time since I've thought about this, but I believe this was a test case I added to catch a bug I introduced (and fixed) while rejiggering `visitedFilter` / `alreadyVisited`.